src/pages/new-chat/new-chat.ts

```diff
@@ -1,6 +1,9 @@
 import { Component, OnInit } from '@angular/core';
-import { MeteorObservable, ObservableCursor } from 'meteor-rxjs';
 import { NavController, ViewController, AlertController } from 'ionic-angular';
+import { Contacts, Contact, ContactFieldType } from 'ionic-native';
+import { MeteorObservable, ObservableCursor } from 'meteor-rxjs';
+import { Observable, Subscription } from 'rxjs';
+import { _ } from 'meteor/underscore';
```

Comment:
Meteor is already loaded with `underscore` so I'd rather avoid the use of a very similar utility library.